src/NzbDrone.Common/Extensions/StringExtensions.cs

```diff
@@ -92,5 +92,13 @@ public static string WrapInQuotes(this string text)
 
             return "\"" + text + "\"";
         }
+
+        public static byte[] HexToByteArray(this string input)
```

Comment:
I know it's the top answer on stackoverflow, but using linq for this kind of stuff? Yuck.